nexus/types/src/deployment.rs

```diff
@@ -1480,6 +1570,184 @@ impl fmt::Display for BlueprintArtifactVersion {
     }
 }
 
+#[derive(
+    Clone,
+    Debug,
+    PartialEq,
+    Eq,
+    Hash,
+    PartialOrd,
+    Ord,
+    JsonSchema,
+    Deserialize,
+    Serialize,
+    Diffable,
+)]
+pub struct BlueprintSingleMeasurement {
+    pub version: BlueprintArtifactVersion,
+    pub hash: ArtifactHash,
+    pub prune: bool,
```

Comment:
(sorry, rust-analyzer is failing me here)

I expected this to just be an ArtifactHash.  Am I understanding correctly that `version` is just here for nicer human-readable output?

What's `prune` for?  I can't find a use of it besides serialization and display.  Maybe it's used in the bigger PR for decision making?  If so, maybe add a comment here?